app/command_groupmsg_test.go

```diff
@@ -10,35 +10,44 @@ import (
-		t.Fatal("error parsing empty message")
-	}
-	if users, parsedMessage := groupMsgUsernames("test"); len(users) != 1 || parsedMessage != "" {
-		t.Fatal("error parsing simple user")
-	}
-	if users, parsedMessage := groupMsgUsernames("test1, test2, test3 , test4"); len(users) != 4 || parsedMessage != "" {
-		t.Fatal("error parsing various users")
-	}
-
-	if users, parsedMessage := groupMsgUsernames("test1, test2 message with spaces"); len(users) != 2 || parsedMessage != "message with spaces" {
-		t.Fatal("error parsing message")
-	}
-
-	if users, parsedMessage := groupMsgUsernames("test1, test2 message with, comma"); len(users) != 2 || parsedMessage != "message with, comma" {
-		t.Fatal("error parsing messages with comma")
-	}
-
-	if users, parsedMessage := groupMsgUsernames("test1,,,test2"); len(users) != 2 || parsedMessage != "" {
-		t.Fatal("error parsing multiple commas in username ")
-	}
-
-	if users, parsedMessage := groupMsgUsernames("    test1,       test2     other message         "); len(users) != 2 || parsedMessage != "other message" {
-		t.Fatal("error parsing strange usage of spaces")
-	}
-
-	if users, _ := groupMsgUsernames("    test1,       test2,,123,@321,+123"); len(users) != 5 || users[0] != "test1" || users[1] != "test2" || users[2] != "123" || users[3] != "321" || users[4] != "+123" {
-		t.Fatal("error parsing different types of users")
-	}
+	assert := assert.New(t)
+
+	users, parsedMessage := groupMsgUsernames("")
+	assert.Len(users, 0)
+	assert.Equal(parsedMessage, "", "error parsing empty message")
+
+	users, parsedMessage = groupMsgUsernames("test")
+	assert.Len(users, 1)
+	assert.Equal(parsedMessage, "", "error parsing simple user")
+
+	users, parsedMessage = groupMsgUsernames("test1, test2, test3 , test4")
+	assert.Len(users, 4)
+	assert.Equal(parsedMessage, "", "error parsing various users")
+
+	users, parsedMessage = groupMsgUsernames("test1, test2 message with spaces")
+	assert.Len(users, 2)
+	assert.Equal(parsedMessage, "message with spaces", "error parsing message")
+
+	users, parsedMessage = groupMsgUsernames("test1, test2 message with, comma")
+	assert.Len(users, 2)
+	assert.Equal(parsedMessage, "message with, comma", "error parsing messages with comma")
+
+	users, parsedMessage = groupMsgUsernames("test1,,,test2")
+	assert.Len(users, 2)
+	assert.Equal(parsedMessage, "", "error parsing multiple commas in username")
+
+	users, parsedMessage = groupMsgUsernames("    test1,       test2     other message         ")
+	assert.Len(users, 2)
+	assert.Equal(parsedMessage, "other message", "error parsing strange usage of spaces")
+
+	users, _ = groupMsgUsernames("    test1,       test2,,123,@321,+123")
+	assert.Len(users, 5)
+	assert.Equal(users[0], "test1")
+	assert.Equal(users[1], "test2")
+	assert.Equal(users[2], "123")
+	assert.Equal(users[3], "321")
+	assert.Equal(users[4], "+123")
+	assert.Equal(parsedMessage, "other message", "error parsing strange usage of spaces")
```

Comment:
This line looks like it's [repeated from the test just before](https://github.com/mattermost/mattermost-server/pull/12175/files#diff-54e30e28064feaf5d69d3e3612115290R43) and not belonging to this one, is that right?

Reply:
yes, Sorry for that mistake.

Reply:
No worries! Thanks for the fixes!! :)

Reply:
@mgdelacroix hi, anything else to do from my side ?